front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -228,9 +230,9 @@ class IdeasNewPage extends PureComponent<Props & WithRouterProps, State> {
         } catch (error) {
           const apiErrors = get(error, 'json.errors');
           if (process.env.NODE_ENV === 'development') console.log(error);
-          if (apiErrors && !apiErrors.idea) {
+
+          if (apiErrors && apiErrors.image) {
```

Comment:
I would like all these API errors' types be more detailed. They're hard to check this way and I think I spotted a mistake here.